duckduckgo/Android
# Feature/david/journal mode

Task/Issue URL: https://app.asana.com/0/1157893581871903/1200123654543284

**Description**:
Background
Change the journal type from write ahead logging to delete. 
This means we won't have to close the database hence avoiding all the threading issues. 
Estimate: Couple of days
This approach is much faster and solves the main issue at hand, thus the preferred approach

**Steps to test this PR**:
Download apk from Jenkins https://jenkins.duckduckgo.com/job/android%20-%20duckduckgo%20app%20release/255/

1. Open app 
2. Get apk contents
3. Ensure app.db journal file does not exist
4. Open a tab
5. Use the Fire button
6. Download app.db file
7. Open with a text editor
8. Check that there is no trace of the previously opened tab


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
